src/build.rs

```diff
@@ -105,8 +105,13 @@ pub struct Build {
     pub no_default_features: bool,
 
     /// Build for the target triple
-    #[clap(long, value_name = "TRIPLE", env = "CARGO_BUILD_TARGET")]
-    pub target: Option<String>,
+    #[clap(
+        long,
+        value_name = "TRIPLE",
+        env = "CARGO_BUILD_TARGET",
+        multiple_occurrences = true
+    )]
+    pub target: Option<Vec<String>>,
```

Comment:
I think `Vec<String>` is enough?

Reply:
But ain't it optional since not providing a target defaults to the host machine?

Reply:
• Pushed a fix